docs/topics_5/web-view-results.adoc

```diff
@@ -3,9 +3,25 @@
 [id='view_results_{context}']
 = Viewing analysis results
 
-Once you have executed {ProductShortName} analysis on a project, the analysis is listed in a table that provides the analysis ID, status, and date started. Each row includes actions available for that analysis, including viewing the analysis details and generated reports, as well as canceling or deleting the analysis.
+The results of all analyses are grouped and listed by project on the *Analysis results* screen.
 
-.Analysis list
-image::web-analysis-list.png[Analysis list]
+.Analysis results screen
```

Comment:
Move this screen shot to step 1 in the procedure. Remember to put a `+` before it so that it indents. You don't need a caption.